packages/fast-components-styles-msft/src/checkbox/index.ts

```diff
@@ -81,6 +82,7 @@ const styles: ComponentStyles<ICheckboxClassNameContract, IDesignSystem> = {
         display: "inline-block",
         width: toPx(20),
         height: toPx(20),
+        alignSelf: "center",
```

Comment:
We shouldn't need to do this in multiple places, the parent can ensure both of these align to one another.